docs/user_guide/notes.rst

```diff
@@ -0,0 +1,34 @@
+Notes for Administrators
+========================
+Make sure to completely read and understand the following before putting ``ngshare`` into production
+
+Race Condition
+--------------
+ngshare should NOT run concurrently, or there may be race conditions and data may be corrupted. For example, do not create multiple ngshare instances that share a same underlying database.
+
+Database Upgrade
+----------------
+ngshare checks database version every time it starts up. If database version is older than ngshare version, it performs schema and data migration. Normally this may only happen after ngshare is updated.
+
+The check can be disabled using ``--no-upgrade-db`` but do not disable it unless you have a good reason and know the possible consequences. 
+
+The database update history can be found in :doc:`/contributer_guide/alembic`.
+
+Database Backup
+---------------
+ngshare users should regularly backup the database. Especially the database should be backed up before updating ngshare. Because after the update, the schema and data migration may corrupt the database.
```

Comment:
```suggestion
ngshare users should regularly back up the database, especially before updating ngshare. The schema and data migration may corrupt the database.
```

Reply:
@aalmanza1998 @Lawrence37 I am changed it to (2 paragraphs):

ngshare users should regularly back up the database in case of corruption.

The database should be backed up before updating ngshare because the schema and data migration may corrupt the database.